kmesh_compile.sh

```diff
@@ -1,22 +1,35 @@
 #!/bin/bash
 
 function prepare() {
-    docker pull ghcr.io/kmesh-net/kmesh-build:v0.2.0
+    local arch
+    arch=$(get_arch)
+    docker pull "ghcr.io/kmesh-net/kmesh-build-${arch}:v0.2.0"
```

Comment:
The compilation image should be relatively stable, and the image tag should be latest.

Reply:
> The compilation image should be relatively stable, and the image tag should be latest.

The compilation image will change with the changes in the software packages that kmesh depends on, and different versions of the compilation image can be used to compile different versions of kmesh.